handlers/main.yml

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: MIT
+---
+# handlers file for nbde_client.
+- name: update initramfs
```

Comment:
I think the handler name is in the global namespace, so it should have the role name in it e.g. `name: nbde_client update initramfs` - @pcahyna I don't think we mention this in the best practices, so perhaps I'm wrong

Reply:
Updating this as well.

Reply:
@richm I think we have this in the best practices ...

Reply:
Sorry, we don't. At least not in the published version.